blockchain/chain_sync/src/sync_worker.rs

```diff
@@ -91,15 +91,15 @@ where
         // Bootstrap peers before syncing
         // TODO increase bootstrap peer count before syncing
         const MIN_PEERS: usize = 1;
-        loop {
-            let peer_count = self.network.peer_manager().len().await;
-            if peer_count < MIN_PEERS {
-                debug!("bootstrapping peers, have {}", peer_count);
-                task::sleep(Duration::from_secs(2)).await;
-            } else {
-                break;
-            }
-        }
+        // loop {
+        //     let peer_count = self.network.peer_manager().len().await;
+        //     if peer_count < MIN_PEERS {
+        //         debug!("bootstrapping peers, have {}", peer_count);
+        //         task::sleep(Duration::from_secs(2)).await;
+        //     } else {
+        //         break;
+        //     }
+        // }
```

Comment:
Also, I'll give more thought on this soon, but probably fine for now to not have the bootstrap wait and to keep this out. I'll try to think of a better long term solution though

Reply:
Cool. One solution i was thinking of was having a reserved sync_worker just for sync submit block. Then we can have an `if` around the loop to indicate is a block from a miner essentially.

Reply:
I don't think that's a great solution, and would turn out fairly clunky. I haven't moved around this bootstrap check since the sync worker has been moved out, is it possible to have this check around the `ChainSyncer` so that it doesn't cue initial sync tipsets until after bootstrap, but workers can still be triggered through RPC? 

I think the fact that this is checked here is just because this used to be all connected and not detached.

Reply:
Hmm yeah... Ill  leave this commented out on this PR and add a todo for now

Reply:
Sounds good, but yeah thinking about it more, it for sure should just happen in the queueing of initial syncs in the `ChainSyncer`